types/sanctuary/index.d.ts

```diff
@@ -97,15 +97,16 @@ declare namespace Sanctuary {
     };
     is(typeRep: TypeRep): (x: any) => boolean;
     //  Showable
-    toString(x: any): string;
+    show(x: any): string;
     //  Fantasy Land
     equals<A>(x: Setoid<A>): (y: Setoid<A>) => boolean;
     lt <A>(x: Ord<A>): (y: Ord<A>) => boolean;
     lte<A>(x: Ord<A>): (y: Ord<A>) => boolean;
     gt <A>(x: Ord<A>): (y: Ord<A>) => boolean;
     gte<A>(x: Ord<A>): (y: Ord<A>) => boolean;
```

Comment:
Should we replace `Ord<A>` with `A` here too?

Reply:
that makes sense, I will check if there are other points where we can make this optimization